apps/molgenis-components/src/Interfaces/IColumn.ts

```diff
@@ -3,7 +3,8 @@ import { ILocale } from "./ILocales";
 export interface IColumn {
   columnType: string;
   id: string;
-  name: string;
+  label: string;
+  description?: string;
   computed?: string;
   conditions?: string[];
   descriptions?: ILocale[];
```

Comment:
All the Ilocale thing can be removed from here right?

Reply:
yes. Irrelevant note: What we might need to have later is two labels, i.e. like 'tableLabel' vs 'formLabel' => the last thing could be called 'formQuestion' for questionnaire like data collection.